scripts/mkcloud

```diff
@@ -1381,6 +1409,10 @@ function sanity_checks
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
 
+    if [[ $want_ceph = 1 ]] && [[ $want_external_ceph = 1 ]] ; then
+        complain 118 "\$want_ceph and \$want_external_ceph are mutully exclusive. Please choose one or the other"
+    fi
```

Comment:
Why not using just one variable here? Eg.
`$want_ceph=1` is deploying it the old way and
`$want_ceph=external` is deploying the new way.

Reply:
hmm, I'd have to think about that. Not saying it isn't a bad idea, want with 0 or >0 seem to be the common way from what I can see.

But would clean up a bit. Let me push the other fixes first as the end of Friday is quickly approaching :)